Sources/CapacitorPluginTools/CapacitorPluginPackage.swift

```diff
@@ -86,4 +86,21 @@ public class CapacitorPluginPackage {
 
         return URL(filePath: fileName, directoryHint: .notDirectory, relativeTo: basePathURL)
     }
+    
+    public func updatePackageJSON() throws {
+        guard let oldPlugin else { throw CapacitorPluginError.oldPluginMissing }
```

Comment:
should this throw a different error? or not throw this in here? it's already being thrown on `findSwiftPluginFile()`

Reply:
Yeah, good point - it should probably never happen, but I do need to check - but they don't charge us by the enum case and more data is better.

Reply:
anyway, `oldPlugin.capacitorPlugin.identifier` is not the correct value, so you can remove the check and use `podspec.podName` instead, but you'll need to modify `updatePackageJSON` to allow the name as param.

Reply:
yeah, that's what I said, `oldPlugin.capacitorPlugin.identifier` is not the correct value to use, but `podspec.podName` since `oldPlugin.capacitorPlugin.identifier` is the plugin/target name, but`podspec.podName` is the target/library name and that's the one needed for xcodebuild to run

It was probably "working" before, but not after this https://github.com/ionic-team/capacitor-plugin-converter/pull/14

Reply:
yeah, fixed it.